Shared/Samples/Run valve isolation trace/RunValveIsolationTraceView.Model.swift

```diff
@@ -0,0 +1,414 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
```

Comment:
A model object shouldn't need to import SwiftUI.

```suggestion
import Combine
import Foundation
```

Reply:
I'll do a clean up of the previous misuses in the future https://github.com/Esri/arcgis-maps-sdk-swift-samples/projects/1#card-89559142.